libs/ledgerjs/packages/cryptoassets/src/data/evm/11155111/erc20-signatures.json

```diff
@@ -0,0 +1 @@
+""
```

Comment:
Why is it needed ?

Reply:
Files are automatically generated by the [evmImporter.js](https://github.com/LedgerHQ/ledger-live/blob/27eb1fb12a70a37536a30de228d0e4046133d819/libs/ledgerjs/script/crypto-assets-importer/evm/erc20Importer.js) file. Also it's important for the file to exist as it will be uploaded to a CDN, and emtpy string != null in that case for the HTTP request response